src/main/java/core/basesyntax/MyHashMap.java

```diff
@@ -1,19 +1,172 @@
 package core.basesyntax;
 
+import java.util.ArrayList;
+import java.util.Arrays;
+import java.util.Collection;
+import java.util.List;
+import java.util.Objects;
+
 public class MyHashMap<K, V> implements MyMap<K, V> {
+    private static final int DEFAULT_INITIAL_CAPACITY = 16;
+    private static final float DEFAULT_LOAD_FACTOR = 0.75f;
+    private static final int DOUBLE_SIZE = 2;
+    private Node<K,V> [] table;
+    private int size;
+    private int threshold;
+
+    public MyHashMap() {
+        table = new Node[DEFAULT_INITIAL_CAPACITY];
+        threshold = (int) (DEFAULT_LOAD_FACTOR * table.length);
+    }
 
     @Override
     public void put(K key, V value) {
-
+        resize();
```

Comment:
Private methods and classes should be after public ones in your class. The 'resize' method should be moved after the public methods.